stabilizer/src/main/java/com/hazelcast/stabilizer/test/utils/TestUtils.java

```diff
@@ -114,6 +115,15 @@ public static void assertTrueEventually(AssertTask task) {
         assertTrueEventually(task, ASSERT_TRUE_EVENTUALLY_TIMEOUT);
     }
 
+    public static void assertEqualsByteArray(byte[] expected,byte[] current){
+        if(!(expected.length == current.length)){
```

Comment:
I did not find alternative way in junit. I found some solutions but they are not i expected.
http://stackoverflow.com/questions/7305612/junit-arrays-not-equal-test